lib/checks/aria/required-children.js

```diff
@@ -13,17 +13,17 @@ function owns(node, role, ariaOwned) {
 	}
 
 	selector = selector.join(',');
-
-	return ariaOwned ? (matchesSelector(node, selector) || !!node.querySelector(selector)) :
-		!!node.querySelector(selector);
+	return ariaOwned ? (matchesSelector(node, selector) || !!axe.utils.querySelectorAll(virtualTree, selector)[0]) :
+		!!axe.utils.querySelectorAll(virtualTree, selector)[0];
 }
 
 function ariaOwns(nodes, role) {
 	var index, length;
 
 	for (index = 0, length = nodes.length; index < length; index++) {
 		if (nodes[index] === null) { continue; }
-		if (owns(nodes[index], role, true)) {
+		let virtualTree = axe.utils.getFlattenedTree(nodes[index]);
```

Comment:
That's the whole problem though–I don't have access to the virtualNode at that time, and the arguments for `getNodeFromTree` include `vNode, node`. This method is called recursively on a collection of elements from `idRefs`, which don't return virtualNodes. There's a comment in that function about only working with the light DOM. How would I get the virtualNode in this `ariaOwns` function without getting the flattened tree first?